### DISCUSSION TITLE
Why does my AirConnect package stop on Synology DSM?

### DISCUSSION THREAD

**Discussion by Yoda50** [eizedev/AirConnect-Synology#50] [Q&A]
I use the latest AirConnect package on the lastest Synology DSM 7. Unfortunately, the AirConnect package stops every day for some strange reason. I have to log on to DSM, go to the Package Center and execute it again - more or less every new day.

Could someone help troubleshooting, please?

**Comment by eizedev** [comment]
Hi @Yoda50 

A sudden stop of the package could have some reasons.  
Could you please share the logfile content of the airconnect log? You can find information of the location of the logfile in the [documentation](https://github.com/Yoda50/AirConnect-Synology#logfiles).

You can use the Filestation app, the share from your windows/linux/mac device or SSH to access the log file.

**Reply by Yoda50** [reply]
Sure thing, @eizedev! Here is my log: http://gofile.me/2Q7wo/UrHlPOlGD Thank you for looking into this issue for me!

**Reply by eizedev** [reply]
Hi @Yoda50 Could you please test the latest release? (1.0.13)

**Reply by Yoda50** [reply]
Before I try again, I have to admit I'm confused, which package is the appropriate one for my new 420+ model (Intel Celeron J4025 Dual Core). You cou please confirm the correct package before I try?

**Reply by eizedev** [reply]
The 420+ has a cpu from the geminilake arch: https://kb.synology.com/en-us/DSM/tutorial/What_kind_of_CPU_does_my_NAS_have

The x86_64 package is the right one: https://github.com/eizedev/AirConnect-Synology/releases/download/1.0.13-20221216/AirConnect-dsm7-x86_64-1.0.13-20221216.spk

**Reply by Yoda50** [reply]
Thanks very much. I've reinstalled. I will test again.
